syntax = "proto3";

service Greeter {
  rpc SayHello (HelloRequest) returns (HelloReply) {}
}

message HelloRequest {
  string name = 1; // 编号为1
}

message HelloReply {
  string message = 1;
}